src/material/list/selection-list.ts

```diff
@@ -46,12 +46,6 @@ export class MatSelectionListChange {
   constructor(
     /** Reference to the selection list that emitted the event. */
     public source: MatSelectionList,
-    /**
-     * Reference to the option that has been changed.
-     * @deprecated Use `options` instead, because some events may change more than one option.
-     * @breaking-change 12.0.0
-     */
-    public option: MatListOption,
```

Comment:
Point for discussion: not sure whether this is a breaking change. Technically we already removed this a couple of major versions ago.

Reply:
Yeah, I wasn't sure either (see issue comment). I would be good trying but at the same time we could just wait until next major.

Reply:
I think its ok to not treat as breaking. Nobody should be using it since it was already removed, and though it was accidentally briefly added back it _was_ still marked as deprecated.